src/uu/pgrep/src/process.rs

```diff
@@ -350,33 +350,21 @@ impl Hash for ProcessInformation {
 
 /// Parsing `/proc/self/stat` file.
 ///
-/// In some case, the first pair (and the only one pair) will contains whitespace,
-/// so if we want to parse it, we have to write new algorithm.
-///
 /// TODO: If possible, test and use regex to replace this algorithm.
 fn stat_split(stat: &str) -> Vec<String> {
     let stat = String::from(stat);
 
-    let mut buf = String::with_capacity(stat.len());
-
-    let l = stat.find('(');
-    let r = stat.find(')');
-    let content = if let (Some(l), Some(r)) = (l, r) {
-        let replaced = stat[(l + 1)..r].replace(' ', "$$");
+    if let (Some(l), Some(r)) = (stat.find('('), stat.rfind(')')) {
```

Comment:
```suggestion
    if let (Some(left), Some(right)) = (stat.find('('), stat.rfind(')')) {
```

I think `left` instead of `l` would be better.

Reply:
Modified :)